app/lib/features/space/pages/shell_page.dart

```diff
@@ -1,26 +1,15 @@
-import 'package:acter/common/providers/room_providers.dart';
-import 'package:acter/common/themes/app_theme.dart';
-import 'package:acter/common/utils/routes.dart';
-import 'package:acter/common/widgets/default_button.dart';
-import 'package:acter/common/widgets/default_dialog.dart';
-import 'package:acter/features/space/model/keys.dart';
 import 'package:acter/features/space/widgets/shell_header.dart';
+import 'package:acter/features/space/widgets/space_toolbar.dart';
 import 'package:acter/features/space/widgets/top_nav.dart';
-import 'package:acter/common/providers/space_providers.dart';
-import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart' show Space;
-import 'package:atlas_icons/atlas_icons.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
-import 'package:go_router/go_router.dart';
 
 class SpaceShell extends ConsumerStatefulWidget {
   final String spaceIdOrAlias;
```

Comment:
without the "child" this isn't really no "shell" anymore now, is it?!? And it isn't a page either. I'd suggest we moved the file from `pages` to `widgets` and name it a `SpaceHeaderWidget` or something.